packages/rules.lua

```diff
@@ -31,23 +67,80 @@ local function registerCommands (_)
     })
   end, "Draws a blob of ink of width <width>, height <height> and depth <depth>")
 
+  SILE.registerCommand("hrulefill", function(options, _)
```

Comment:
```suggestion
  SILE.registerCommand("hrulefill", function (options, _)
```

As a minor style point and 100% my preference, not a requirement, I try to format function *definitions* with a space after the function keyword and function *calls* without, so

```lua
-- define
local foo = function (bar) end
-- call
foo(bar)
```

Similarly I keep stuff in parens tight (`foo(6)`) but stuff in braces loose (`foo({ bar = 6 })`). Again a super minor issue but I do periodically normalize all SILE's code to this convention to make it easier to search for things & use my own editor shortcuts ;-)